spec/function.dd

```diff
@@ -2749,17 +2749,56 @@ $(H3 $(LNAME2 trusted-functions, Trusted Functions))
 
         $(P Trusted functions are marked with the $(CODE @trusted) attribute.)
 
-        $(P Trusted functions are guaranteed by the programmer to not exhibit
-        any undefined behavior if called by a safe function.
-        Generally, trusted functions should be kept small so that they are
-        easier to manually verify.
+        $(P Trusted functions are guaranteed to not exhibit any undefined
```

Comment:
The old text "by the programmer" was adequate, no? Put another way: "Trusted functions are assumed without checking that ..."

Reply:
The old text was fine. I just moved the "by the programmer" part to a new sentence, because I'm adding another requirement, and repeating "by the programmer" didn't look nice to me.

If you think "assumed without checking" is better than "guaranteed", I can change that. I don't see what difference it makes, though.

Reply:
"guaranteed" by itself implies guaranteed by the compiler. Guaranteed by something else must be explicitly stated.

Reply:
> "guaranteed" by itself implies guaranteed by the compiler. Guaranteed by something else must be explicitly stated.

It's stated later in the paragraph. Good enough?